tools/rvn/CommandLineApp.cs

```diff
@@ -144,6 +153,7 @@ private static void ConfigureSetupPackage()
                     var packageOutPathVal = packageOutPath.Value();
                     var certPathVal = certPath.Value();
                     var certPassTuple = certPass.Value() ?? Environment.GetEnvironmentVariable("RVN_CERT_PASS");
+                    var generateHelmValuesVal = generateHelmValues.HasValue() && generateHelmValues.Value() is null ? "values.yaml" : generateHelmValues.Value(); 
```

Comment:
And if it does not have a value we'll still try to do `.Value()`?

Reply:
it'll return `null` and the code will still work